pkg/controller/status/status.go

```diff
@@ -200,7 +200,12 @@ func (c *Controller) merge(existing *configv1.ClusterOperator) *configv1.Cluster
 	if len(c.namespace) > 0 {
 		existing.Status.RelatedObjects = []configv1.ObjectReference{
 			{Resource: "namespaces", Name: c.namespace},
-			{Group: "apps", Resource: "deployments", Namespace: c.namespace, Name: "insights-operator"},
+			{Resource: "secrets", Namespace: "openshift-config", Name: "pull-secret"},
+			{Resource: "secrets", Namespace: "openshift-config", Name: "support"},
```

Comment:
This one is kinda optional, but we try to read it so I think it makes sense to include it.